VideoAPI/VideoApi.AcceptanceTests/Features/Events.feature

```diff
@@ -5,7 +5,25 @@
 
 Scenario: Add conference event
 	Given I have a conference
-    And I have a valid conference event request for event type Joined
-    When I send the request to the endpoint
-    Then the response should have the status NoContent and success status True
+	And I have a valid conference event request for event type Joined
```

Comment:
please stop checking in white spaces

Reply:
I don't know that do you mean, have we any standards for formatting?